src/web/lyd_og_bilde/README.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "Lærerveiledning - HTML: Legg til lyd og video"
+level: 3
+language: nb
+---
+
+# Om oppgaven {.activity}
+I denne oppgaven skal elevene lære å legge til lyd og video på en nettside. I oppgaven blir det presentert hvordan man legger til video fra _Youtube_, _Vimeo_ og video man har liggende lokalt på datamaskinen. Det blir videre presentert forskjellige måter å legge opp lyd på.
+
+
+## Oppgaven passer til: {.check}
+ __Fag__: Programmering, Teknologi i praksis, Norsk
+
+__Anbefalte trinn__: 7.-10. trinn
+
+__Tema__: Web, HTML, lyd, film
+
+__Tidsbruk__: Dobbeltime
+
+
+## Kompetansemål {.challenge}
```

Comment:
Musikk, 10.trinn: "gjøre rede for regler for opphavsrett knyttet til bruk av musikk"?